src/server/components/HtmlLayout.jsx

```diff
@@ -1,13 +1,20 @@
 import { PropTypes } from 'react';
 
 const propTypes = {
-    head: PropTypes.object.isRequired,
+    head: PropTypes.shape({
+        title: PropTypes.shape({ toComponent: PropTypes.func.isRequired }).isRequired,
```

Comment:
Not very nice, but it is the signature of react-helmet head. What were you thinking ?